tsconfig.base.json

```diff
@@ -9,6 +9,7 @@
     "skipLibCheck": true,
     "baseUrl": ".",
     "paths": {
+      "@ory/nextjs": ["packages/nextjs/src/index.ts"],
```

Comment:
Don't think this is needed with npm workspaces, why did you add it?

Reply:
Done!